docs/docs/jobs.md

```diff
@@ -125,6 +125,21 @@ for job in jobs:
     print('Job %s: %s' % (job.id, job.func_name))
 ```
 
+#### Job Status
+
+The status of a job can be one of the following:
+
+* `queued`: The default status for created jobs, except for those that have dependencies, which will be created as `deferred`. These jobs are ready to be executed.
+* `finished`: The job has finished execution and is available through the finished job registry.
+* `failed`: Jobs that encountered errors during execution or expired before being executed.
+* `started`: The job has started execution. This status includes the job execution support mechanisms, such as setting the worker name and setting up heartbeat information.
+* `deferred`: The job is not ready for execution because its dependencies have not finished successfully yet.
+* `scheduled`: Jobs created to run at a future date or jobs that are retried after a retry interval.
+* `stopped`: The job was stopped because the worker was stopped, and will be re-queued.
+* `canceled`: The job has been manually canceled and is not runnable or inspectable, even if it is part of a dependency set.
```

Comment:
`has been manually canceled and will not be executed, even if it is part of a dependency chain.`